fenix/app/src/main/java/org/mozilla/fenix/compose/DropdownMenu.kt

```diff
@@ -76,6 +85,33 @@ fun DropdownMenu(
     }
 }
 
+/**
+ * Dropdown menu for presenting context-specific actions.
+ *
+ * @param menuItems List of items to be displayed in the menu.
+ * @param showMenu Whether or not the menu is currently displayed to the user.
+ * @param onDismissRequest Invoked when user dismisses the menu or on orientation changes.
+ * @param modifier Modifier to be applied to the menu.
+ * @param offset Offset to be added to the position of the menu.
+ */
+@Composable
+fun ContextualDropdownMenu(
```

Comment:
```suggestion
fun ContextDropdownMenu(
```

Reply:
Oh I see it's called Contextual in the figma. Maybe we should be trying to sway @cwzilla into renaming it to Context dropdown menu.

Reply:
Perhaps `ContextualMenu` for the time being?

Reply:
`ContextualMenu` works for me, and I agree with keeping it succinct.